app/src/main/res/layout-land/lessons_chapter_view.xml

```diff
@@ -37,26 +37,24 @@
 
     <TextView
       android:id="@+id/chapter_index"
+      style="@style/Subtitle2"
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
       android:layout_marginStart="4dp"
-      android:fontFamily="sans-serif"
       android:importantForAccessibility="@{viewModel.chapterPlayState != ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES ? View.IMPORTANT_FOR_ACCESSIBILITY_YES : View.IMPORTANT_FOR_ACCESSIBILITY_NO}"
       android:text="@{viewModel.computePlayChapterIndexText()}"
-      android:textColor="@{viewModel.chapterPlayState != ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES ? @color/oppiaPrimaryText : @color/oppiaPrimaryText30}"
-      android:textSize="14sp" />
+      android:textColor="@{viewModel.chapterPlayState != ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES ? @color/oppiaPrimaryText : @color/oppiaPrimaryText30}" />
 
     <TextView
       android:id="@+id/chapter_name"
+      style="@style/Subtitle2"
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
```

Comment:
Remove `layout_width` and `layout_height` attribute as it is already mentioned in `Body` style. (Make sure its `wrap_content` only in styles.xml)